applications/luci-app-unbound/luasrc/model/cbi/unbound.lua

```diff
@@ -125,6 +131,15 @@ wfq:value("4", translate("Interface FQDN, All Addresses"))
 wfq:depends({ dhcp_link = "none" })
 wfq:depends({ dhcp_link = "odhcpd" })
 
+exa = s1:taboption("advanced", ListValue, "add_extra_dns", translate("Extra DNS:"),
+  translate("Use extra DNS entries found in /etc/config/dhcp"))
+exa:value("0", translate("Do Not Use"))
+exa:value("1", translate("Use Menu Network/Hostnames"))
+exa:value("2", translate("Use MX/SRV RR (no LuCI)"))
```

Comment:
Working on words that will fit in the list boxes without expanding overlay presentation and make sense.